### DISCUSSION TITLE
how to use body_block

### DISCUSSION THREAD

**Discussion by casualmatt** [rspamd/rspamd#4286] [Q&A]
Hello, 

I don't find a lot of documentation on the body_block option, I intend to rewrite part of the mail body to remove 1pixel trackers. 

Is that possible?

In general how to use body_block in a Lua script and what is the replay to the request?

**Comment by vstakhov** [comment]
Do you use milter or HTTP protocol?

**Reply by casualmatt** [reply]
Rspamd module for haraka, i suppose http
